source/_integrations/seventeentrack.markdown

```diff
@@ -74,3 +74,13 @@ content: >
 ```
 
 {% endraw %}
+
+In a templated Markdown card of this type, the following attributes are available for each individual package:
```

Comment:
That is not specific for the markdown card, but for the entity state.
Maybe consider adding a chapter that describes the state and its attributes.